Dockerfile.ubuntu-arm64

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:22.04 AS build
```

Comment:
Touched on this in Slack - we should build in AL2
(and even if we do use Ubuntu, it should be the lowest version possible in order to support the most distros & versions we can, not 22.04 which is the latest LTS).

Given the trouble you've already experienced with cross compiling in Linux containers and the consideration that we'll manually build a Linux arm64 build on AL2 when releasing anyway, I'd rather we _don't_ include an automated Ubuntu arm64 build at all.
I'm a little concerned that including this here would incentivise whoever's releasing to skip building & testing an AL2 arm64 build properly.